pkg/sql/conn_executor_exec.go

```diff
@@ -271,6 +271,7 @@ func (ex *connExecutor) execStmtInOpenState(
 	}
 
 	ex.incrementStartedStmtCounter(ast)
+	ex.incrementStartedQueryCounter()
```

Comment:
Could we reuse the `incrementStartedStmtCounter` method to do the work that you want `incrementStartedQueryCounter` to do? We're actually recording stats for all statements, not just queries (usually "query" just means DML), and it seems unnecessary to have 2 calls that are always called together.

Reply:
Done.
Moved the `updateRollingQueryCounts` method that was being called in `incrementedStartedQueryCounter` to `incrementStartedStmtCounter`.

Removed `incrementedStartedQueryCounter`.